docs/howto/configurations-ref.rst

```diff
@@ -0,0 +1,947 @@
+
```

Comment:
Can you add page title e.g. "Configuration Reference"?  The title is very important because it is used to generate links in menu.